### DISCUSSION TITLE
The Outlook 1px "line" issue

### DISCUSSION THREAD

**Discussion by JennBohm** [foundation/foundation-emails#1126] [Q&A]
Just in case anyone is not familiar, there is an issue with Outlook for Windows where there are seemingly random 1px "lines" that appear in emails. It seems they are actually tiny gaps that are showing the body background, and I believe they have to do with rounding involved with the px to pt conversion that the Word rendering engine does. Based on [this discussion](https://social.technet.microsoft.com/Forums/ie/en-US/1f4f1e03-8ad5-4079-bc18-8fcaaa7b785d/outlook-2016-renders-horizontal-lines-seemingly-randomly-in-html-emails), it appears that there is no sure solution, and it is an Outlook bug. So, definitely not saying anything is wrong with Foundation :) 

What I'm wondering is, has anyone else here faced these issues, and how are they handling them within the context of Foundation emails? 

What I have tried is making all font sizes and such in even numbers (I still need to get some logo images to have even px heights and widths, but waiting on our client's designers to provide them) and applying the appropriate background image to every row/column/other element within the section that has the line showing up. 

I've seen some more obscure sounding suggestions, like inserting non-breaking space just before closing </td> tags, but I haven't tried that, because I'm coding in Inky, and all the actual table tags are automatically generated.

**Comment by elcaptain** [comment]
Sorry for this unhelpful answer, but I've just learned to live with it and ignore the lines ^^"

**Comment by scottbickell** [comment]
I have seen these lines in Foundation emails as well as others. To add to the strangeness, sometimes moving the email window from one display (a high resolution retina to a non-retina or vice versa) to another can eliminate the lines. This layout bug is a known issue in Outlook. I've spent many hours trying resolve it, now I just ignore it. Not worth the effort
